src/Sherpa/ArticleViewController.swift

```diff
@@ -168,4 +190,12 @@ internal class ArticleViewController: ListViewController {
 		}
 	}
 	
+    fileprivate func _applyStyle(toString string: String) -> String {
+        
+        let styled = "<style>body{ font-family: 'HelveticaNeue', 'Arial', 'Serif'; font-size: 44px; background-color: #efeff4 }</style><body>\(string)</body>"
```

Comment:
Ideally, the `background-color` specified here should be transparent, or determined programatically based on the value of the table view's `backgroundColor` property. The latter is customisable using  `UIAppearance` and having a hard-coded value here will be a problem when this happens.